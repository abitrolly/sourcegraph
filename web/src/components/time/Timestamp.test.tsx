import React from 'react'
import { Timestamp } from './Timestamp'
import { mount } from 'enzyme'

describe('Timestamp', () => {
    test('mocked current time', () => expect(mount(<Timestamp date="2006-01-02" />)).toMatchSnapshot())

    test('noAbout', () => expect(mount(<Timestamp date="2006-01-02" noAbout={true} />)).toMatchSnapshot())
})
