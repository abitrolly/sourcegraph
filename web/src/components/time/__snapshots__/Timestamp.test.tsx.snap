// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timestamp mocked current time 1`] = `
<Timestamp
  date="2006-01-02"
>
  <span
    className="timestamp"
    data-tooltip="2006-01-02"
  >
    about 22 hours ago
  </span>
</Timestamp>
`;

exports[`Timestamp noAbout 1`] = `
<Timestamp
  date="2006-01-02"
  noAbout={true}
>
  <span
    className="timestamp"
    data-tooltip="2006-01-02"
  >
    22 hours ago
  </span>
</Timestamp>
`;
